control-plane/modules/aws-k3s/files/k3s-install.sh

```diff
@@ -6,6 +6,9 @@ export INSTALL_K3S_VERSION='v${install_k3s_version}'
 export INSTALL_K3S_EXEC='%{ if is_k3s_server }${k3s_tls_san} ${k3s_disable_agent} ${k3s_deploy_traefik} %{~ endif ~}${k3s_exec}'
 export K3S_CLUSTER_SECRET='${k3s_cluster_secret}'
 export K3S_DATASTORE_ENDPOINT='${k3s_datastore_endpoint}'
+export CATTLE_BOOTSTRAP_PASSWORD="admin"
```

Comment:
This should be a templated value

Reply:
Actually this isnt meant to be there at all, I forgot to push it up.

Reply:
Removed